helpers/pluck.js

```diff
@@ -1,8 +1,21 @@
 'use strict';
 
+const helpers = require('handlebars-helpers');
+
 const factory = () => {
     return function(collection, path) {
-        return collection.map(item => item[path]);
+        if (collection === null || typeof collection === 'undefined' || typeof collection !== 'object') {
+            return '';
+        }
+
+        let res = [];
```

Comment:
picked up this logic from here https://github.com/helpers/handlebars-helpers/blob/master/lib/array.js#L445

Reply:
@junedkazi So, why we don’t just grab "pluck" from handlebars-helpers? (and similarly, we can do with other utils)